Sources/ExistentialFuture.swift

```diff
@@ -61,6 +61,17 @@ private final class FilledFuture<Value>: FutureBase<Value> {
     }
 }
 
+// Concrete future wrapper that will never get filled.
+private final class UnfilledFuture<Value>: FutureBase<Value> {
```

Comment:
Not unreasonable, since it's private. Will probably do the same to `FilledFuture` - `Box`.